index.js

```diff
@@ -121,6 +132,12 @@ var publicAgent = {
     var headers = {};
     headers[constants.TRACE_AGENT_REQUEST_HEADER] = 1;
 
+    if (modulesLoadedBeforeTrace.length > 0) {
```

Comment:
We don't have the ability to log before this point. We precompute as much of the information as possible before `start` but need to wait until a logger is initialized to warn.